### ISSUE TITLE
TypeError: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by turribeach** [FlyingDiver/Indigo-BMW-CD#4]
Hi, I am getting the following error randomly:

```
BMW Connected Drive Debug       ConnectedDrive update_vehicles`
BMW Connected Drive Debug       ConnectedDrive get_vehicle_data: xxxxxxxxxxxxxxxx`
BMW Connected Drive Error       Error in plugin execution runConcurrentThread:`

Traceback (most recent call last):`
 File "plugin.py", line 140, in runConcurrentThread`
 File "plugin.py", line 179, in updateVehicle`
 File "plugin.py", line 21, in dict_to_states`
TypeError: 'NoneType' object is not iterable`

 BMW Connected Drive Error       plugin runConcurrentThread function returned or failed (will attempt again in 10 seconds)`
```
Seems to be happening when it tries to do get_vehicle_status. I think it's a problem on the BMW API side since the error comes and goes. Possibly they are having server issues as I saw this error too once:

```
  BMW Connected Drive Error       ConnectedDrive Account Update failed, response = '{"error":{"code":500,"description":"(SmartPhoneUtil-A-302) External service failed. Cannot get services\nby: (SmartPhoneUtil-E-302) External service failed. Failed to call service: Hystrix command SvdsGetServices failed. Error when calling SVDS. Received Status 503"}}'
```
But I thought in reporting it in case others see it too.

**Comment by FlyingDiver** [comment]
The second is definitely a server error.  The first might be, but I shouldn't be throwing an exception when it happens.  I'll need to fix that, at least.

**Comment by FlyingDiver** [comment]
Fixed in 0.0.7
